purchases/src/test/java/com/revenuecat/purchases/paywalls/PaywallDataTest.kt

```diff
@@ -74,14 +77,51 @@ class PaywallDataTest {
             assertThat(darkColors.accent2?.stringRepresentation).isEqualTo("#FF00FF")
         }
 
-        mockLocale("en_US", "eng")
-        mockLocale("es_ES", "spa")
         mockLocale("gl_ES", "glg")
 
         val requiredLocale = Locale("gl_ES")
         assertThat(paywall.configForLocale(requiredLocale)).isNull()
     }
 
+    @Test
+    fun `finds locale if it only has language`() {
+        val json = File(javaClass.classLoader!!.getResource(PAYWALLDATA_SAMPLE1).file).readText()
```

Comment:
This is repeated in every test, what do you think about extracting it like `loadJSON(PAYWALLDATA_SAMPLE1)`?